arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -58,5 +58,16 @@ public async Task PutCollectionTruncateAsync(string collectionName)
             }
             await GetApiErrorException(response);
         }
+
+        public async Task<GetCollectionCountResponse> GetCollectionCountAsync(GetCollectionCountOptions options = null)
+        {
+            var response = await _transport.GetAsync(_collectionApiPath + "/" + options.CollectionName + "/count");
```

Comment:
In #25 @rossmills99 wraps the response object inside a `using` statement. Should we do the same for new client methods?

Reply:
Thanks for the question, but it could be even more useful if you gave your opinion rather than just pose the question.  What do you think?  Do you have a recommendation here?

Reply:
Well a using statement limits the scope of the response variable within a method. These api methods are not terribly complex. So I don't think it is necessary.

Reply:
The point of `using` is not to limit scope, but to ensure `Dispose` gets called on the disposable object instantiated in the `using` declaration.  I think as a rule we _should_ make sure the underlying `HttpResponseMessage` instance gets disposed.

Reply:
I also think it's good to use `using` because right now we don't expose anything that is still referenced in the response object.